cmd/jaeger/badger_config.yaml

```diff
@@ -0,0 +1,43 @@
+service:
+  extensions: [jaeger_storage]
+  pipelines:
+    traces:
+      receivers: [otlp, jaeger, zipkin]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]
+
+extensions:
+  jaeger_query:
+    trace_storage: memstore
+    ui_config: ./config-ui.json
+
+  jaeger_storage:
+    badger:
+      memstore:
```

Comment:
give better name, e.g. badger_primary

Reply:
sure!

Reply:
not what I meant - the "type" of storage was correctly named `badger` (this would be the name of the property in the config), but the "name" of that storage instance needs to be something else, not "memstore".